src/TagLib/CombinedTag.cs

```diff
@@ -1270,8 +1270,50 @@ public override string[] PerformersRole
 					if (tag != null)
 						tag.MusicBrainzArtistId = value;
 			}
-		}
-
+		}
+
+		/// <summary>
+		///    Gets and sets the MusicBrainz Release Group ID.
+		/// </summary>
+		/// <value>
+		///    A <see cref="string" /> containing the MusicBrainz
+		///    ReleaseGroupID for the media described by the 
+		///    current instance or null if no value is present.
+		/// </value>
+		/// <remarks>
+		///    <para>When getting the value, the child tags are looped
+		///    through in order and the first non-<see langword="null" />
+		///    and non-empty value is returned.</para>
+		///    <para>When setting the value, it is stored in each child
+		///    tag.</para>
+		/// </remarks>
+		/// <seealso cref="Tag.MusicBrainzReleaseGroupId" />
+		public override string MusicBrainzReleaseGroupId
+		{
+			get
+			{
+				foreach (Tag tag in tags)
+				{

```

Comment:
I think this is the last nitpick about formatting. The opening curly brace should be on the same line. ie,
`
		public override string MusicBrainzReleaseGroupId {
			get {
				foreach (Tag tag in tags) {
`